.changeset/cyan-timers-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+'faustwp': minor
```

Comment:
While we are removing functionality, it seems like it's functionality that was never exposed to the user. Thoughts on making this a `patch` release?